helm-charts/powerdns/templates/configmap.yaml

```diff
@@ -46,6 +47,7 @@ data:
     {{ $key }}={{ $value }}
     {{ end }}
     forward-zones={{ tpl .Values.powerdns.recursor.forward_zones .}}
+    local-address={{ .Values.global.provisioningIP }}:53
```

Comment:
missing the global definition in values.yaml?

Reply:
I made this optional now,